### PR TITLE
$$$ ~ Miner Swag ~ $$$

### PR DESCRIPTION
**Pull request by fosskers**
This PR improves swagger documentation related to the new `/mining/` endpoints.

### TODO

- [x] Hand-written `ToSchema` instance for `Miner`
- [x] Improved field names for `Miner`
- [x] Other instances

### CODE REVIEW

File: src/Chainweb/RestAPI/Orphans.hs [hunk 1]
```diff
@@ -431,17 +431,28 @@ deriving instance ToSchema a => ToSchema (Time a)
 deriving instance ToSchema a => ToSchema (TimeSpan a)
 
 instance ToSchema ChainwebVersion where
-  declareNamedSchema _ = pure $ NamedSchema (Just "ChainwebVersion") mempty
+    declareNamedSchema _ = pure $ NamedSchema (Just "ChainwebVersion") mempty
```
Comment (mightybyte) [inline src/Chainweb/RestAPI/Orphans.hs]: Please revert these indentation changes.  It makes things harder to review.

File: src/Chainweb/Miner/Pact.hs [hunk 2]
```diff
@@ -89,17 +89,20 @@ data Miner = Miner !MinerId !MinerKeys
     deriving stock (Eq, Ord, Show, Generic)
     deriving anyclass (NFData)
 
+-- NOTE: These JSON instances are used (among other things) to embed Miner data
+-- into the Genesis Payloads. If these change, the payloads become unreadable!
+--
```
Comment (fosskers) [inline src/Chainweb/Miner/Pact.hs]: It may be policy if this instance can be changed or not. Internally then old field names are more compact/efficient, but externally the UX is bad. Both `chainweb-node` and `chainweb-miner` utilize these instances for configuration.

File: test/Chainweb/Test/Pact/PactExec.hs [hunk 3]
```diff
@@ -143,14 +144,14 @@ testReq6 = TestRequest
 -- -------------------------------------------------------------------------- --
 -- Utils
 
-execTest :: (forall a . (PactDbEnv' -> PactServiceM cas a) -> IO a) -> TestRequest -> ScheduledTest
+execTest
+    :: (forall a . (PactDbEnv' -> PactServiceM cas a) -> IO a)
+    -> TestRequest
+    -> ScheduledTest
```
Comment (fosskers) [inline test/Chainweb/Test/Pact/PactExec.hs]: The linting in these test files is drive-by cleanups done while investigating a bug.

File: src/Chainweb/RestAPI/Orphans.hs [hunk 4]
```diff
@@ -431,17 +432,46 @@ deriving instance ToSchema a => ToSchema (Time a)
...
-  declareNamedSchema _ = pure $ NamedSchema (Just "MerkleLogHash") mempty
+    declareNamedSchema _ = pure $ NamedSchema (Just "MerkleLogHash") mempty
 
--- TODO Need more detail for these two!
 instance ToSchema HeaderBytes where
-  declareNamedSchema _ = pure $ NamedSchema (Just "HeaderBytes") mempty
+    declareNamedSchema _ = pure . NamedSchema (Just "HeaderBytes") $ binarySchema
+        & description ?~ "An encoded BlockHeader"
+        & minLength ?~ 302
+        & maxLength ?~ 302
```
Comment (larskuhtz) [inline src/Chainweb/RestAPI/Orphans.hs]: This number depends on the chain web version. While the chainweb version is part of the API type, it's not available here.

Comment (fosskers) [reply src/Chainweb/RestAPI/Orphans.hs]: I got `302` from the Wiki. Is it still a safe default for now?

Comment (larskuhtz) [reply src/Chainweb/RestAPI/Orphans.hs]: It's valid for Peterson graph and twenty chain (both 3 regular) with 32 bit hashes.

Comment (larskuhtz) [reply src/Chainweb/RestAPI/Orphans.hs]: So, currently it holds for devnet and testnet
